cfme/services/myservice/ssui.py

```diff
@@ -59,6 +59,9 @@ def is_displayed(self):
     configuration = VersionPick({
         Version.lowest(): SSUIConfigDropdown("dropdownKebabRight"),
         '5.8': SSUIDropdown('Configuration')})
+    lifecycle_btn = VersionPick({
```

Comment:
**Required**

This duplicates the widget on L55, and please take this opportunity to change to `lifecycle` instead of `lifecycle_btn` - especially since it isn't a button.  We want to make this change to all the toolbar widgets, removing `_btn` suffix.

Reply:
Done